homepage/templates/homepage/landing/landing.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>ClassRater</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">

```

Comment:
the meta tags as well are present in app_layout, so they can also be safely erased from this file.
sorry I wasn't clear in the previous comment